pkg/model/firewall.go

```diff
@@ -73,14 +75,71 @@ func (b *FirewallModelBuilder) buildNodeRules(c *fi.ModelBuilderContext) error {
 		c.AddTask(t)
 	}
 
-	// Nodes can talk to master nodes
-	{
-		t := &awstasks.SecurityGroupRule{
-			Name:          s("all-node-to-master"),
+	// TODO: We need to remove the ALL rule
+	//W1229 12:32:22.300132    9003 executor.go:109] error running task "SecurityGroupRule/node-to-master-443" (9m58s remaining to succeed): error creating SecurityGroupIngress: InvalidPermission.Duplicate: the specified rule "peer: sg-f6b1a68b, ALL, ALLOW" already exists
+	//status code: 400, request id: 6a69627f-9a26-4bd0-b294-a9a96f89bc46
+
+	udpPorts := []int64{}
+	tcpPorts := []int64{}
+	protocols := []string{}
```

Comment:
Suggest a map here instead of a slice of strings.. that way we can have the protocol name, and a friendly name

```
"4" => "IPIP"
```

So we can name the SG accordingly on line ~138